src/gdclasses/OpenXRConfig.cpp

```diff
@@ -306,3 +310,37 @@ void OpenXRConfig::stop_passthrough() {
 		passthrough_wrapper->stop_passthrough();
 	}
 }
+
+godot::Array OpenXRConfig::get_play_space() {
+	ARVRServer *server = ARVRServer::get_singleton();
+	Array arr;
+	Vector3 sides[4] = {
+		Vector3(-0.5f, 0.0f, -0.5f),
+		Vector3(0.5f, 0.0f, -0.5f),
+		Vector3(0.5f, 0.0f, 0.5f),
+		Vector3(-0.5f, 0.0f, 0.5f),
+	};
+
+	if (openxr_api && server) {
+		Size2 extends = openxr_api->get_play_space_bounds();
+		if (extends.x != 0.0 && extends.y != 0.0) {
+			Transform reference_frame = server->get_reference_frame();
+
+			for (int i = 0; i < 4; i++) {
+				Vector3 coord = sides[i];
+
+				// scale it up
+				coord.x *= extends.x;
+				coord.z *= extends.y;
```

Comment:
Here, better to use `extends.width` and `extends.height` to avoid any confusion when reading the code (e.g: wondering why a seemingly y-axis value is assigned to `z`).

Reply:
Owh I had forgotten there is a union on Vector2 (which Size2 is coming from) exposing x and y as width and height, yeah makes sense!